cointop/colorscheme.go

```diff
@@ -275,13 +275,16 @@ func (c *Colorscheme) Color(name string, a ...interface{}) string {
 	return c.ToSprintf(name)(a...)
 }
 
-func (c *Colorscheme) GocuiFgColor(name string) gocui.Attribute {
-	var attrs []gocui.Attribute
-	if v, ok := c.colors[name+"_fg"].(string); ok {
-		if fg, ok := c.ToGocuiAttr(v); ok {
-			attrs = append(attrs, fg)
-		}
+func (c *Colorscheme) TcellFgColor(name string) tcell.Color {
```

Comment:
Do we need separate FgColor() and BgColor() lookup?  Could combine into a lookup that does the _fg and _bg and returns a Style?

Reply:
I have unified the lookup. Tks!